src/Text/Tabular/Base.agda

```diff
@@ -0,0 +1,166 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Fancy display functions for List-based tables
+--
+-- The functions in this module assume some (unenforced) invariants.
+-- If you cannot guarantee that your data respects these invariants,
+-- you should instead use Text.Tabular.List.
+------------------------------------------------------------------------
+
+{-# OPTIONS --safe --without-K #-}
+
+module Text.Tabular.Base where
+
+open import Data.Bool.Base using (if_then_else_)
+open import Data.Char.Base using (Char)
+open import Data.List.Base as List
+  using (List; []; _∷_; _?∷_; _++_; _∷ʳ?_; null; map; intersperse)
+open import Data.Maybe.Base as Maybe using (Maybe; nothing; just; maybe)
+open import Data.Nat.Base
+open import Data.String.Base as String
+  using (String; fromChar; unlines; replicate; length)
+open import Function.Base
+open import Agda.Builtin.Equality
+
+open String
+  using ( Alignment
+        ; Left
+        ; Center
+        ; Right
+        ) public
+
+record TabularLine : Set where
+  field left  : Maybe String
```

Comment:
Could we align the fields below the `field` keyword in the record as we usually do? Need to add this to the style guide...